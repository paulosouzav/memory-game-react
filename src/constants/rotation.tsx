import { css } from 'styled-components';

export const ROTATION = {
  half: css`
    transform: rotateY(180deg);
  `,
};
