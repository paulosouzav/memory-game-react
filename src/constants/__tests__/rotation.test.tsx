import { ROTATION } from '../rotation';

describe('ROTATION', () => {
  it('should have expected values', () => {
    expect(ROTATION).toMatchSnapshot();
  });
});
