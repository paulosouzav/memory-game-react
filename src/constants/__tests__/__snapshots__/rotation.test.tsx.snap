// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ROTATION should have expected values 1`] = `
Object {
  "half": Array [
    "
    transform: rotateY(180deg);
  ",
  ],
}
`;
